roles/openshift_prometheus/templates/prometheus.j2

```diff
@@ -72,6 +74,7 @@ spec:
       - name: prometheus
         args:
         - --storage.tsdb.retention=6h
+        - --storage.tsdb.min-block-duration=2m
```

Comment:
The oauth-proxy changed, it needs the following flags:

```
      -openshift-ca=/etc/pki/tls/cert.pem
      -openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
```

for both prometheus and alerts proxy

Reply:
I don't see it here:
https://github.com/openshift/origin/blob/master/examples/prometheus/prometheus.yaml

Is that in an unmerged PR?

Reply:
@smarterclayton Is this a must for this PR, or can go on a different one?

Reply:
@zgalor on IRC you stated it works without these settings.  Does it work with them as well?  If so, I do not see the harm in adding them in.

Reply:
@jcantril I didn't test with them yet. I guess if that oauth-proxy changed this would require a version bump, and since I didn't see that on the template, I didn't want to make partial changes before @smarterclayton replies.

But if this is not mandatory for statefulset I would rather make the change in a follow-up pr than block on this.

Reply:
@jcantrill

Reply:
Ok. Arguments were added and retested. Pod gets deployed and running with no errors on the logs.
@jcantrill Please approve.